plugin/trino-pinot/pom.xml

```diff
@@ -587,6 +604,15 @@
                     </rules>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <suiteXmlFiles>
+                        <file>src/test/resources/suite.xml</file>
```

Comment:
https://github.com/trinodb/trino/blob/513d193a2d77799d93358adfc8b5ca2e92b7b410/plugin/trino-sqlserver/pom.xml#L19-L25

is the canonical fix for now until Surefire is updated.

Reply:
This worked, thanks! I used `instances` as the `classes` had the same errors that `methods` had. Let me know if it's ok to squash the commits.

Reply:
Looks good to squash. Thanks.